lib/peer.ml

```diff
@@ -1,26 +1,12 @@
-type state =
+type status =
   | Alive
   | Suspicious
   | Faulty
 
 type t = {
-  address : string;
-  port : int;
-  known_peers : t list;
-  state : state;
+  socket_address : Address.t;
```

Comment:
Maybe just address is fine for the field name

Reply:
Agreed, done.